staging/src/k8s.io/component-base/logs/logs.go

```diff
@@ -176,6 +186,10 @@ func InitLogs() {
 		// Otherwise LoggingConfiguration.Apply will do this.
 		klog.StartFlushDaemon(logFlushFreq)
 	}
+
+	// This is the default in Kubernetes. Options.ValidateAndApply
+	// will override this with the result of a feature gate check.
+	klog.EnableContextualLogging(false)
```

Comment:
Why set this to `false` and not `contextualLoggingDefault`?

Reply:
When we reach beta, contextualLoggingDefault will be true. At that point, we'll probably still want the feature to be off in those commands which don't have a feauture gate parameter because then a beta feature would be permanently enabled for them. That seems too soon, we probably only should do that when the feature is GA.